crates/docs/src/utils.rs

```diff
@@ -0,0 +1,41 @@
+use std::{env, path::PathBuf};
+
+use crate::snippet::Snippet;
+
+#[must_use]
+pub fn get_parent_dir(levels_up: usize) -> PathBuf {
```

Comment:
```suggestion
pub fn get_nth_ancestor(levels_up: usize) -> PathBuf {
```

Or something different but suggesting it's going up by the number of `levels_up`.